src/query/api/v1/handler/prometheus/remote/tag_values.go

```diff
@@ -82,5 +82,9 @@ func (h *TagValuesHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 
 	// TODO: Support multiple result types
-	prometheus.RenderTagCompletionResultsJSON(w, result)
+	err = prometheus.RenderTagValuesResultsJSON(w, result)
```

Comment:
maybe do error check separately ? On calling this function with the io.Writer, it might seem as if you can still have error while something is already written back.

Reply:
It only completes the write in the flush method in the `jw.Close()` call, so if it errors, nothing is written.